qa/rpc-tests/bip68-112-113-p2p.py

```diff
@@ -209,11 +209,8 @@ def get_tests(self):
         self.tip = int("0x" + self.nodes[0].getbestblockhash(), 0)
         self.nodeaddress = self.nodes[0].getnewaddress()
 
-        assert_equal(get_bip9_status(self.nodes[0], 'csv')['status'], 'defined')
         test_blocks = self.generate_blocks(61, 4)
         yield TestInstance(test_blocks, sync_every_block=True) # 1
-        # Advanced from DEFINED to STARTED, height = 143
-        assert_equal(get_bip9_status(self.nodes[0], 'csv')['status'], 'started')
```

Comment:
not clear to me why all this `get_bip9` got removed

Reply:
csv is activated a long time ago so this portion of the test seems unnecessary today.  We can't reuse the csv bit (0), until we get it out of the code and that code is no longer being run.